pkgs/games/gcompris/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, cmake, qtbase, fetchurl, qtdeclarative, qtmultimedia, qttools, qtsensors, qmlbox2d, gettext, qtquickcontrols, qtgraphicaleffects, makeWrapper,
+  gst_all_1
+}:
+stdenv.mkDerivation rec {
+  version = "0.90";
+  name = "gcompris-${version}";
+
+  src = fetchurl {
+    url = "http://gcompris.net/download/qt/src/gcompris-qt-${version}.tar.xz";
+    sha256 = "1i5adxnhig849qxwi3c4v7r84q6agx1zxkd69fh4y7lcmq2qiaza";
+  };
+
+  enableParallelBuilding = true;
+
+  cmakeFlags = "-DQML_BOX2D_LIBRARY=${qmlbox2d}/${qtbase.qtQmlPrefix}/Box2D.2.0";
+
+  nativeBuildInputs = [ cmake makeWrapper ];
```

Comment:
Please add ninja when using cmake.

Reply:
I was not aware of the runtime detection of `ninja` by `cmake`. Thank you. Fixed.